lib/src/messages/pause_collection.dart

```diff
@@ -0,0 +1,18 @@
+part of '../../messages.dart';
+
+@JsonSerializable()
+class PauseCollection extends Message {
+  final PauseCollectionBehavior behavior;
+  final DateTime? resumesAt;
```

Comment:
```suggestion
  @TimestampConverter()
  final DateTime? resumesAt;
```
I guess it won't work without this.